bin/nvidia-container-toolkit

```diff
@@ -26,15 +26,26 @@ device_wait() {
 
 }
 
+# Just exit if NVIDIA support is disabled #
+NVIDIA_SUPPORT_DISABLED="$(snapctl get nvidia-support.disabled)"
```

Comment:
Thinking more about this, this adds a second toggle to turn Nvidia support off. Because, previous pull request already added that (potentially without cleanup) indirectly.

Since nvidia-container-toolkit is now a service, one can do `snap services disable docker.nvidia-container-toolkit` which feels like a natural thing to do, if that is the service one doesn't want from the docker snap.

I don't know if the snap is notified about such change and if we should key on if `snapctl services` is enabled or not.

The changes themselves are good. So could be merged, but I do wonder if we can get away with just service enabled/disabled toggle (default enabled).

Reply:
Yes, I think the problem is, as you already notice, disable does give any usable signal which could trigger a cleanup.